### ISSUE TITLE
Laravel 6.0 Support

### ISSUE THREAD

**Issue by mikemand** [Torann/laravel-geoip#148]
Laravel 6 will release next week. Here's the [upgrade guide](https://laravel.com/docs/6.0/upgrade).

**Comment by voicecode-bv** [comment]
@Torann can you please update GeoIPServiceProvider.php?

```
use Illuminate\Support\Str;

/**
     * Check if package is running under Lumen app
     *
     * @return bool
     */
    protected function isLumen()
    {
        return Str::contains($this->app->version(), 'Lumen') === true;
    }
```

**Comment by Torann** [comment]
Fixed with #150
